client/src/app/components/nova/address/section/AddressPageTabbedSections.tsx

```diff
@@ -88,6 +93,12 @@ const buildAccountAddressTabsOptions = (
         isLoading: isCongestionLoading,
         infoContent: bicMessage,
     },
+    [ACCOUNT_TABS.Validation]: {
+        disabled: !hasValidatorDetails,
```

Comment:
Maybe we should have stricter criteria for "disabled/hidden" for this one, as it seams like all addresses "have validation details". One the ones i checked it looks like this:
![image](https://github.com/iotaledger/explorer/assets/6864498/e93b5749-710c-48f4-8889-8de508f9515f)

Are cases like this something that we want to show ? @begonaalvarezd

Reply:
I dont think it makes sense adding validation tab if the account has no staking feature